openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCard/PopoverContent.tsx

```diff
@@ -69,7 +69,22 @@ const PopoverContent: FC<Props> = ({
     setVisible(newVisible);
   };
 
-  const deleteButtonCheck = threadId && postId && onConfirmation && isAuthor;
+  const deleteButtonCheck = useMemo(() => {
+    const baseCheck = Boolean(threadId && postId && onConfirmation);
+    if (isAnnouncement) {
+      return Boolean(baseCheck && (isAuthor || currentUser?.isAdmin));
+    } else {
+      return baseCheck && isAuthor;
+    }
+  }, [threadId, postId, onConfirmation, isAuthor, currentUser]);
+
+  const editCheck = useMemo(() => {
+    if (isAnnouncement) {
+      return isAuthor || currentUser?.isAdmin;
+    } else {
+      return !isThread && isAuthor;
+    }
+  }, [isAnnouncement, isThread, isAuthor, currentUser]);
```

Comment:
Admin should have access to delete or edit any thread, not just announcements

Reply:
Sure @vivekratnavel .

Reply:
Done.